recipe-server/client/control_new/tests/components/forms/test_FormItem.js

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import FormItem from 'control_new/components/forms/FormItem';
+import WrappedFormItem from 'control_new/components/forms/FormItem';
```

Comment:
See comment re: `static trimValue`, which should let this stay how it was.

Reply:
I don't think that making `trimValue()` static will fix this.

The issue is that the `@connectFormProps` decorator wraps the `FormItem` component, and therefore we can't access anything from `FormItem` without using something like `wrappedComponent`.